apps/src/dance/LottieDancerRenderer.ts

```diff
@@ -0,0 +1,673 @@
+import lottie, {
+  type AnimationItem,
+  type CanvasRendererConfig,
+} from 'lottie-web';
+
+import appConfig from '../music/appConfig';
```

Comment:
Since we're outside of Music Lab, just import `queryParams` from `@cdo/apps/code-studio/utils` directly.